src/libmerklecpp/test/flush.cpp

```diff
@@ -12,24 +12,27 @@
 
 int main()
 {
+  auto test_start_time = std::chrono::high_resolution_clock::now();
+  double timeout = get_timeout();
```

Comment:
That's oddly specific but ok :)

Reply:
How so? It gets the timeout from the environment variable.

Reply:
I meant making it explicitly a double when everything around is auto :)

Reply:
Oh yes, that's unnecessary :)